### ISSUE TITLE
Exasol timestamp format with T separator supported as of version 7 - what should  be default?

### ISSUE THREAD

**Issue by tglunde** [tglunde/dbt-exasol#35]
Exasol supports literal ‚T‘ als separator: https://www.exasol.com/support/browse/EXASOL-2608

Alternatively one could set execute ALTER SESSION/SYSTEM as standard: ```alter session set NLS_TIMESTAMP_FORMAT = 'YYYY-MM-DDTHH:MI:SS';```

Currently, timestamps are rejected by default, since using 'T' is not the default in Exasol. Therefore CSV files for seeding are currently rejected by default. To change this behaviour, we have two ideas: 

a) using preHook in project.yml - needs to be tested
b) using additional parameter in profile.yml to set standard parameters - additionally one could add other language settings according to locale

Any ideas? @ilikutle @jups23 @Fredehagelund92 @sti0

**Comment by sti0** [comment]
Hi @tglunde ,
I don't understand exactly what you mean with "Therefore CSV files for seeding are currently rejected by default." Could you explain further and maybe add an example?
Thanks

**Comment by tglunde** [comment]
Hi,
if you want to import a CSV file using dbt seed with the following content:
```
ID,NAME,SOME_DATE
1,Easton,1981-05-20T06:46:51
2,Lillian,1978-09-03T18:10:33
```
it will fail with an error regarding timestamp format ( [timestamp format](https://en.wikipedia.org/wiki/ISO_8601) ). 

The following CSV file content ist accepted:
```
ID,NAME,SOME_DATE
1,Easton,1981-05-20 06:46:51
2,Lillian,1978-09-03 18:10:33
```

Thats why I think we either add parameters in profile.yml. Alternatively one could add pre-hook. In both cases we would execute corresponding ALTER SESSION commands for the timestamp on the session for the connection. I am just not sure wether those pre-hooks actually fire for seeds. That needs to be tested.

Regards, Torsten

**Comment by tglunde** [comment]
solved by merge request #48
